.github/CODEOWNERS

```diff
@@ -1,2 +1,2 @@
-* @luke-zhu @arthur-observe @stefandorsett
+* @obs-gh-colinhutchinson @stefandorsett
```

Comment:
We should remove Stefan here, he gets a massive amount of noise from things his name is on.

Separately, I have a hard personal rule that I will not add _my_ name to repositories as a CODEOWNER. In general I tend to find the concept of _individual_ "ownership" of software products in a corporate environment to be unmaintainable bordering on toxic. 

> The smallest unit of software ownership isn’t the individual, it’s the *team*. [🐦](https://twitter.com/mipsytipsy/status/1533635174767833088)

I'd always prefer a _team_ (in GitHub) to an individual. This has many benefits, in order of importance:

1. [Automated review assignment](https://docs.github.com/en/organizations/organizing-members-into-teams/managing-code-review-settings-for-your-team). Notifying everyone is assigning no one. Automated review assignment ensures both clear responsibility and fair workloads.
2. It's easy to add members and public repository readers don't see team churn.

Not insisting you do this now but we should tackle this soon, across the board. Part of that is decided what the teams should be. I believe I've been added to a few Terraform/AWS repositories individually.

I made a `terraform-maintainers` team:

https://observeinc.slack.com/archives/C02J6PHLAKD/p1678133630185939

Seems like it would make sense to have teams under integrations for each major cloud:

* aws
* google
* azure

And then we can readily ensure that each has a few members with some but not total overlap and that, for example, all Google repositories have the same underlying owners.